src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity/Protocol.php

```diff
@@ -18,6 +18,7 @@
 
 namespace Surfnet\ServiceProviderDashboard\Domain\Entity\Entity;
 
+use Exception;
```

Comment:
Using a self-made exception is more in line with how we roll in the domain layer. Suggestion: `ProtocolNotFoundException`, `ProtocolMappingFailedException`,...